aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+        </p>
+
+        <pre><code>&lt;h1 id="irc-log">IRC Log&lt;/h1>
+&lt;div role="log" aria-labelledby="irc-log">
+ &lt;p>[10:26] &amp;lt;Charl> ok let&#39;s test it and see if it works&lt;/p>
+ &lt;p>[10:59] &amp;lt;hsivonen> morning&lt;/p>
+&lt;/div>
+</code></pre>
+      </section>
+
+      <section id="live_region_role_status">
+        <h4>Live Region Role <code>status</code></h4>
+
+        <p>
+          The <code>status</code> role indicates that content is advisory information for the user but is not important enough to justify an <code>alert</code>, often but not necessarily presented as a status bar.
+        </p>
+
+        <p>
+          The default value for <code>aria-live</code> is <code>polite</code>.
+          The default value for <code>aria-atomic</code> is <code>true</code>.
+        </p>
+
+        <p>
+          Do not move focus to the element with script when the content is changed.
+        </p>
+
+        <p>
+          For example, the search result summary example above could use <code>role="status"</code> instead of <code>role="region"</code>:
+        </p>
+
+        <pre><code>&lt;form role="search" aria-labelledby="search">
+ &lt;h2 id="search">Search&lt;/h2>
+ &lt;label>Search query: &lt;input type="search" name="q" oninput="updateSearch(event)">&lt;/label>
+ &lt;div id="search-result-status" role="status">&lt;/div>
+&lt;/form>
+</code></pre>
+
+        <p>
+          The HTML <code>output</code> element has the <code>status</code> role by default.
+          The <code>output</code> element is defined to represent the result of a calculation performed by the application, or the result of a user action.
+          The result of a user search is thus appropriate use of the <code>output</code> element:
+        </p>
+
+        <pre><code>&lt;form role="search" aria-labelledby="search">
+ &lt;h2 id="search">Search&lt;/h2>
+ &lt;label>Search query: &lt;input type="search" name="q" oninput="updateSearch(event)">&lt;/label>
+ &lt;output id="search-result-status">&lt;/output>
+&lt;/form>
+</code></pre>
+      </section>
+
+      <section id="live_region_role_timer_and_marquee">
+        <h4>Live Region Role <code>timer</code> and <code>marquee</code></h4>
+
+        <p>
+          Usage of the <code>timer</code> and <code>marquee</code> roles is discouraged.
+        </p>
+
+        <p>
+          The <code>timer</code> role is a numerical counter which indicates an amount of elapsed time from a start point, or the time remaining until an end point.
+          The <code>timer</code> role is a subclass of the <code>status</code> role.
+          The default value of <code>aria-live</code> value of <code>off</code>.
+        </p>
+
+        <p>
+          The <code>marquee</code> role indicates non-essential information that changes frequently.
+          The default value of <code>aria-live</code> value of <code>off</code>.
```

Comment:
Remove redundant "value of".

```suggestion
          The default value of <code>aria-live</code> is <code>off</code>.
```